parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveType.java

```diff
@@ -401,15 +400,18 @@ public PrimitiveType(Repetition repetition, PrimitiveTypeName primitive,
    * @param name the name of the type
    */
   public PrimitiveType(Repetition repetition, PrimitiveTypeName primitive, int length, String name) {
-    this(repetition, primitive, length, name, null, null, null);
+    this(repetition, primitive, length, name, (LogicalTypeAnnotation) null, null, null);
   }
 
   /**
    * @param repetition OPTIONAL, REPEATED, REQUIRED
    * @param primitive STRING, INT64, ...
    * @param name the name of the type
    * @param originalType (optional) the original type to help with cross schema convertion (LIST, MAP, ...)
+   *
+   * @deprecated use {@link #PrimitiveType(Repetition, PrimitiveTypeName, String, LogicalTypeAnnotation)} instead
```

Comment:
The usual pattern in parquet for deprecating is to mention that the related method will be removed in 2.0.0. In case of the suggestion would be to use one of the overloaded methods then it is fine to not mention.